querydsl-sql/src/main/java/com/querydsl/sql/AbstractSQLQuery.java

```diff
@@ -626,10 +626,19 @@ private long unsafeCount() throws SQLException {
         }
     }
 
+    /**
+     * loqQuery is to log queryString and it's parameters
+     * @param queryString, list of parameters
+     * @return void, this is to log input parameters
+     */
     protected void logQuery(String queryString, Collection<Object> parameters) {
         if (logger.isLoggable(Level.FINE)) {
             String normalizedQuery = queryString.replace('\n', ' ');
             logger.fine(normalizedQuery);
+            Object[] parametersArray = parameters.toArray();
+            for (int i = 0; i < parameters.size(); i++) {
+                logger.fine("Query Parameter:" + String.valueOf(i) + "-->" + parametersArray[i].toString());
```

Comment:
Can `parametersArray[i]` be null?
If yes, there will be NPE on `parametersArray[i].toString()`.